recipe/meta.yaml

```
@@ -1,48 +1,56 @@
-{% set name = "casa_formats_io" %}
+{% set name = "casa-formats-io" %}
 {% set version = "0.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace("-", "_") }}-{{ version }}.tar.gz
   sha256: 169423d17799eefbcecd433ffb9a86e85470357977833a14048b58750d4ce26e
 
 build:
-  skip: true  # [py<39]
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 3
+  number: 0
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
-    - {{ stdlib("c") }}
   host:
+    - pip
     - python
+    - numpy >=2.0.0
```

https://github.com/radio-astro-tools/casa-formats-io/blob/v0.3.0/pyproject.toml#L4
```suggestion
    - numpy {{ numpy }}  # [py<39]
    - numpy >=2.0.0  # [py>=39]
```